sass/includes/_buttons.scss

```diff
@@ -8,6 +8,11 @@
   margin: 0.5rem 0;
   border: none;
 
+  &__target-blank-label {
+    display: block;
+    font-size: .7em;
```

Comment:
Hi @AshTNA - would this be better as `rem` rather than `em`?

Reply:
Thanks @gtvj  - I've made this change